### PR TITLE
Add halborn security headers

### PR DESCRIPTION
**Pull request by DmitriyMolch**
https://cerenetwork.atlassian.net/browse/CBI-2265

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -2,6 +2,9 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
+    <meta http-equiv="Content-Security-Policy" content="default-src 'self'; connect-src ws://localhost:3000 http://localhost:3000 wss://*.cere.network https://*.infura.io https://*.google-analytics.com wss://*.walletconnect.org https://*.walletconnect.com https://*.sentry.io; style-src 'self' 'unsafe-inline'; img-src 'self' data:; script-src 'self' 'unsafe-inline' 'unsafe-eval' https://*.googletagmanager.com">
```
Comment (AndreiNavoichyk) [inline public/index.html:5]: Why do we need `ws://localhost:3000`?

Comment (DmitriyMolch) [reply public/index.html:5]: I'm getting an error in the console without it
image

Comment (DmitriyMolch) [reply public/index.html:5]: I see `ws` connection in the console.
image

Comment (AndreiNavoichyk) [reply public/index.html:5]: Do you need to add `ws` to DEV, STAGE, PROD?

Comment (DmitriyMolch) [reply public/index.html:5]: It's not required there.

Comment (DmitriyMolch) [reply public/index.html:5]: I already released this branch on dev yesterday and there are no errors there.

**Review by AndreiNavoichyk — APPROVED** [review]

**Pull request merged**